rdr_service/dao/bq_questionnaire_dao.py

```diff
@@ -48,12 +48,21 @@ def make_bqrecord(self, p_id, module_id, latest=False, convert_to_enum=False):
         # Query to get a list of questionnaire_response_id values for this participant and module
         # The list will be from most recently received/replayed response to earliest.  This mirrors how the
         # deprecated stored procedure sp_get_questionnaire_answers used to order its results
+        #
+        # PDR-254:  Updating the module response data to include a test_participant flag as a common field
+        # Intended to make filtering module response data by test_participant status more efficient in BigQuery
         _participant_module_responses_sql = """
             select qr.questionnaire_id, qr.questionnaire_response_id, qr.created, qr.authored, qr.language,
-                   qr.participant_id, qh2.external_id, qr.status
+                   qr.participant_id, qh2.external_id, qr.status,
+                   CASE
+                       WHEN p.is_test_participant = 1  or p.is_ghost_id = 1 or h.name = :test_hpo THEN 1
+                       ELSE 0
+                   END as test_participant
```

Comment:
Note:  I believe RDR `participant` table is now a reliable source of truth in using the ghost / test participant / HPO pairing details to determine test participant status, and would have already identified test participants and paired them to TEST based on their recognizable test email or login phone number.